schema.json

```diff
@@ -140,6 +140,10 @@
             "type": "string",
             "description": "e.g. XYZ Institute of Technology - [Add institute name]"
           },
+          "location": {
```

Comment:
For the location currently, we are following the below-mentioned format:
https://github.com/resumic/schema/blob/master/schema.json#L54-#L68

Reply:
Thanks, I'll change it to be more in line with that format